core/src/main/java/org/ehcache/EhcacheManager.java

```diff
@@ -129,6 +135,8 @@ void closeEhcache(final String alias, final Ehcache<?, ?> ehcache) {
     Class<K> keyType = config.getKeyType();
     Class<V> valueType = config.getValueType();
     
+    LOGGER.info("Cache is getting created in EhcacheManager .");
```

Comment:
I would add the alias to the log statement, multiple caches can be created in parallel.